lte/gateway/c/sctpd/sctpd_downlink_impl.cpp

```diff
@@ -19,13 +19,15 @@
 #include <unistd.h>
 #include "sctpd.h"
 #include "util.h"
-
+#include "magma_logging.h"
```

Comment:
Not needed here as it is already included in "util.h"

Reply:
okay